spatial-history/index.md

```diff
@@ -1,39 +1,120 @@
+<blockquote>
+<em>
+“What results is a steady stream of well-trained engineers, business people, marketers, researchers; a vibrant venture capital community; a highly available stock market appetite for stock flotations; and people with experience in business, including how and why business failures happen.” [<a href="http://www.bbc.com/news/technology-26041341">1</a>]
+</em>
+</blockquote>
+In his interview, he discussed that a stream of well-trained workers is one of the key components to Silicon Valley’s success. University of Toronto is known for its prestigious education. University of Toronto’s Engineering, Computer Science, and Rotman School of Management program is ranked number one in Canada, and is well known for their research. OCAD is the fourth largest art school in North America and the oldest in Canada [<a href="https://en.wikipedia.org/wiki/Education_in_Toronto">10</a>]. Students around the world come here to learn about art. Infamous companies such as Google and Amazon already has their eye on Toronto. Google searched through cities across North America, Europe and Australia to build their high-tech neighbourhood or their “high-tech Quayside” [<a href="https://www.thestar.com/news/city_hall/2017/10/17/google-firm-wins-competition-to-build-high-tech-quayside-neighbourhood-in-toronto.html">3</a>]. Not only did Google find Toronto to be a great location for their innovation to start, Amazon is searching for their second headquarters. Amazon has included Toronto in their top 20 city list out of 238 cities that requested to be the home of Amazon’s HQ2 [<a href="https://www.amazon.com/b?ie=UTF8&node=17044620011">4</a>]. It is the only city to make the list in Canada. Toronto is the perfect location with infrastructure that produces the most talented workers and already acknowledged by the infamous tech companies.
+</p>
+
+<p style="text-align: right;">
+Toronto's future high-tech neighbourhood [<a href="https://www.thestar.com/news/city_hall/2017/10/17/google-firm-wins-competition-to-build-high-tech-quayside-neighbourhood-in-toronto.html">3</a>].
+<img style="width:60%;" src="https://www.thestar.com/content/dam/thestar/news/city_hall/2017/10/17/google-firm-wins-competition-to-build-high-tech-quayside-neighbourhood-in-toronto/_4_sidewalk_labs.jpg.size.custom.crop.850x601.jpg" />
+</p>
+
+<p>
+Location is a key component to a company's success. The area must have a high population with the right resources and a strong network. Toronto is the most populated city in Canada. It is the home to 4,753,120 people, making it the seventh largest city in North America [<a href="http://canada.businesschief.com/top10/1003/Canadas-Top-10-Most-Populous-Cities">2</a>]. It may not be the largest city in the world, however, it is the world’s most diverse city in the world [<a href="http://www.metronews.ca/news/toronto/2016/05/16/toronto-the-diverse.html">5</a>].
+
+</br>
+<figure style="text-align: center;">
```

Comment:
this is the kind of occasion when it's worth using HTML instead of markdown. Try to limit to these kinds of very specific cases.